p/api/greader.php

```diff
@@ -1116,8 +1116,7 @@ function markAllAsRead($streamId, $olderThanId) {
 		case 'unread-count':
 			$output = isset($_GET['output']) ? $_GET['output'] : '';
 			if ($output !== 'json') notImplemented();
-			$all = isset($_GET['all']) ? $_GET['all'] : '';
-			unreadCount($all);
+			unreadCount();
```

Comment:
unreadCount didn't do anything with the parameter?

Reply:
it has no parameter

https://github.com/FreshRSS/FreshRSS/blob/ba48733f17006b5e07625a22811594e62033fcc8/p/api/greader.php#L443-L446